pkg/logstash.sysv

```diff
@@ -141,7 +141,6 @@ configtest() {
     return 1
   fi
 
-  JAVA_OPTS=${LS_JAVA_OPTS}
   HOME=${LS_HOME}
   export PATH HOME JAVA_OPTS LS_HEAP_SIZE LS_JAVA_OPTS LS_USE_GC_LOGGING
```

Comment:
but then why do we need to export it here? I understand we need to export it if we actually redefined itself here.

the options you are talking about are exported in https://github.com/elastic/logstash/blob/master/bin/logstash.lib.sh#L90, I don't see why se need to export it here too, is redundant to my understanding.